WebRoot/js/zimbraMail/share/model/ZmZimletBase.js

```diff
@@ -1399,4 +1399,3 @@ function() {
 	var c = this.xmlObj("contentObject");
 	return (c && c.toolTip && c.toolTip.sticky && c.toolTip.sticky.toLowerCase() === "true");
 };
-
```

Comment:
nit: please update it so that the file does not appear in the diff.

Reply:
Done